src/components/filters/Chip/Chip.css

```diff
@@ -1,6 +1,6 @@
 .g3-chip {
   box-sizing: border-box;
-  font-size: 14px;
+  font-size: 1em;
```

Comment:
why not `0.875rem` for this one?

Reply:
true, that's a mistake

Reply:
and i don't have precommit hooks installed, i'll add them

Reply:
ok installed. including proof for posterity's sake lol 
<img width="629" alt="Screen Shot 2021-06-25 at 8 54 35 AM" src="https://user-images.githubusercontent.com/7152954/123435399-27d34c80-d593-11eb-94a0-76ec90ba7589.png">